roles/kubernetes/kubeadm/handlers/main.yml

```diff
@@ -1,5 +1,15 @@
 ---
-- name: restart kubelet
+- name: Kubeadm | restart kubelet
+  command: /bin/true
+  notify:
+    - Kubeadm | reload systemd
+    - Kubeadm | reload kubelet
```

Comment:
Since Ansible 2.2 you can use `listen` on handler instead of this `/bin/true` hack

Reply:
thanks